drupal/code/modules/features/kada_configuration_feature/kada_configuration_feature.features.field_base.inc

```diff
@@ -75,13 +75,11 @@ function kada_configuration_feature_field_default_field_bases() {
     'module' => 'list',
     'settings' => array(
       'allowed_values' => array(
-        'lime' => 'Lime',
-        'mint' => 'Mint',
+        'red' => 'Red',
         'orange' => 'Orange',
-        'pink' => 'Pink',
-        'gray' => 'Gray',
-        'blue' => 'Blue',
         'yellow' => 'Yellow',
```

Comment:
We need an exception for the yellow color so that the text color is always black when the background is yellow. Now there is too little contrast between the colors.

Reply:
Added an if into sass to set text-color to black when background is yellow.